en/concepts/scene/light/lightmap.md

```diff
@@ -1,12 +1,14 @@
 # Lightmapping
 
-> **Note**: the Lightmapping feature is not currently supported on Apple M1 (Silicon) architecture devices and is expected to be supported in v3.1.2.
+> **Note**: the Lightmapping feature is not currently supported on Apple M1 (Silicon) architecture devices and is expected to be supported in v3.2.1.
 
-Lightmaps are generated by the **Baking System** by calculating the effect of all light sources in the scene on the model. In scenes with fixed light sources, using lightmapping instead of real-time lighting calculations can reduce resource consumption and thus improve the efficiency of scene operation.
+The **Baking System** pre-calculates the lighting, shadows, etc. that a static object with a stable light source will receive. The result of this calculation is stored in a texture map, which we call a **light map**.
+
+Cocos Creator automatically processes and uses the generated light map at runtime. In scenes with fixed light sources, using light maps instead of real-time lighting calculations can reduce resource consumption and thus increase the efficiency of the scene.
```

Comment:
```suggestion
Cocos Creator automatically processes and uses the generated light map at runtime. In scenes with fixed lights, using light maps instead of real-time lighting calculations can reduce resource consumption and thus increase the efficiency of the scene.
```